components/browser/session/src/main/java/mozilla/components/browser/session/Session.kt

```diff
@@ -51,6 +51,7 @@ class Session(
         fun onDesktopModeChanged(session: Session, enabled: Boolean) = Unit
         fun onFullScreenChanged(session: Session, enabled: Boolean) = Unit
         fun onThumbnailChanged(session: Session, bitmap: Bitmap?) = Unit
+        fun onLoadComplete(session: Session, url: String) = Unit
```

Comment:
I don't see a difference between this and `loading = false`.

I also think this breaks the contract of Session: It is expected to encapsulate the state and makes it observable. This just propagates an event and no state is saved.